plasmapy/analysis/swept_langmuir/tests/test_helpers.py

```diff
@@ -0,0 +1,71 @@
+"""Tests for `plasmapy.analysis.swept_langmuir.helpers`."""
+
+import numpy as np
+import pytest
+
+from contextlib import ExitStack as does_not_raise
+
+from plasmapy.analysis.swept_langmuir.helpers import check_sweep
+
+# TODO: replace ExitStack with nullcontext when we require >= python 3.7
```

Comment:
This makes me want to require python >= 3.7 even though I don't know what `ExitStack` and `nullcontext` are.

Reply:
It's a conditional for `with` the does nothing.  It's useful when you have a test function that mixes cases that raises and does not raise `Exception`'s/`Warning`'s.  For example,

```python
@pytest.mark.parametrize(
    "args, expected, conditional",
    [
        (1, None, pytest.mark.raises(PlasmaPyError)),
        ("hello", "goodbye", ExitStack()),
    ],
)
def test_something(args, expected, conditional):
    with conditional:
        results = foo(*args)
        assert results == expected
```

Reply:
Oh!  Thanks!